src/odemis/driver/smaract.py

```diff
@@ -1046,7 +1046,7 @@ class MC_5DOF_DLL(CDLL):
 
     # property symbols
     SA_MC_PKEY_PIVOT_POINT_MODE = 0x00001001
```

Comment:
BTW, the include file has two extra properties:
```
#define SA_MC_PKEY_MODEL_CODE                           0x0a02
#define SA_MC_PKEY_MODEL_NAME                           0x0a03
```
The smaract_mc private bindings for python also mention a few extra keys, including 
`    VERSION_STRING                    = 0xf001`

Could you have a look at reading this properties, to update hwVersion and swVersion with useful info?